src/styles/blocks/product-card.css

```
@@ -0,0 +1,91 @@
+.product-card {
+  font-family: "Roboto", sans-serif;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.product-card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  font-style: normal;
+
+  margin-bottom: 4px;
+}
+
+.product-card__product-number {
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  margin-bottom: 16px;
+}
+
+.product-card__raing {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.product-card__rewiews {
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.product-card__price-line {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.product-card__price-icon-text {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+}
+
+.product-card__price {
+  color: #060b35;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.product-card__button {
+  display: inline-block;
+  width: 100%;
+  padding: 12px 0;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: solid 1px transparent;
+
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+}
+
+.product-card__button:hover {
+  border: 1px solid #00acdc;
```

You can change just border-color
```suggestion
  border-color: #00acdc;
```

---

fixed